js/timex.js

```diff
@@ -979,6 +979,8 @@ module.exports = class timex extends Exchange {
             'type': undefined,
             'name': name,
             'active': active,
+            'deposit': undefined,
+            'withdraw': undefined,
```

Comment:
Both `deposit` and `withdraw` are actually defined, take a look at the response sample above.